src/ocean/io/select/EpollSelectDispatcher.d

```diff
@@ -300,6 +317,36 @@ public class EpollSelectDispatcher : IEpollSelectDispatcherInfo
         }
     }
 
+    /***************************************************************************
+
+        Registers a cycle hook that will be executed when the select cycle
+        finished.
+
+        Params:
+            hook = delegate to be called after the select cycle finished
+
+    ***************************************************************************/
+
+    public void cycleHook (bool delegate() hook)
```

Comment:
Either use the type alias here i.e.
```d
public void cycleHook (selectCycleHook hook)
```
or don't use a type alias at all (my personal preference).